pommerman/utility.py

```diff
@@ -328,3 +331,43 @@ def get_next_position(position, direction):
 
 def make_np_float(feature):
     return np.array(feature).astype(np.float32)
+
+def join_json_state(record_json_dir, agents, finishedAt, config):
+    jsonSchema = {
+        "properties": {
+            "state": {
+                "mergeStrategy": "append"
+            }
+        }
+    }
+
+    jsonTemplate = {
+        "agents": agents,
+        "finishedAt": finishedAt,
```

Comment:
s/finishedAt/finished_at

Reply:
cool. an old habit of working with mongo.